RTN-002.tex

```diff
@@ -57,66 +59,71 @@
+The Community Science Team (CST) in the Rubin System Performance department maintains this set of user profiles and use cases
+to capture the range of anticipated needs regarding scientific analyses with the Rubin software and data products.
+
+This document guides the development of a process to scientifically validate user-facing resources for scientific analysis,
+such as the Rubin Science Platform (RSP), documentation, tutorials, and activities.
+To ``scientifically validate" the RSP means to confirm that the RSP's functionality meets the scientific needs
+of its users, and that users are able to do their science with the RSP's tools.
+This document also guides the development of Rubin's Model for Community Science \citedsp{RTN-004}.
 
 \textbf{User profiles} are designed to represent the diverse user community of the RSP:
 different experience and expertise backgrounds, scientific motivations, and accessibility needs.
-Four user profiles for each of the Notebook, Portal, and API aspects of the RSP have been drafted along
-the axes of user experience and science motivation.
-Two additional user profiles have been drafted: one to represent integrated use of the Notebook and Portal aspects,
-and one to represent accessibility needs.
 
-These fourteen user profiles are subject to change.
+\textbf{Analysis use cases} are designed to represent the variety of types of analysis that
+users will do with the RSP, such as subsetting, visualization, model fitting, and reprocessing.
+These use cases help to define the limitations of the RSP as well, and identify analysis
+that requires external resources (independent data access centers; IDACs).
 
-\textbf{Use cases} for the support of scientists using the Rubin software and data products have been drafted
-in a format defined by model-based systems engineering (MBSE) practices.
-The seven use cases represent issues and support needs that involve, e.g., trouble with scientific analysis,
+\textbf{Support use cases} are designed to provide workflow examples of Rubin's issue resolution
+process, to illuminate how users report problems and get support, and how staff from across
+Rubin participate in user support.
+These use cases include, e.g., trouble with scientific analysis,
 instrumentation faults, software bugs.
 
-These seven use cases are subject to change.
-
 \clearpage
-\section{User Profiles for Science Validation of the Rubin Science Platform}
+\section{User Profiles}
 
-\clearpage
-\subsection{Portal Aspect}
-\input{up_portal.tex}
+\input{user_profiles.tex}
 
-\clearpage
-\subsection{Notebook Aspect}
-\input{up_notebook.tex}
 
 \clearpage
-\subsection{Web Application Programming Interface (API) Aspect}
-\input{up_web_api.tex}
+\section{Analysis Use Cases}
 
-\clearpage
-\subsection{Integrated Aspects}
-\input{up_integrated_aspects.tex}
+These analysis use cases represent general things that users want to do in the Rubin Science Platform,
+using specific scientific examples.
```

Comment:
The analysis use case need to represent general things users want to do in the RSP, and should roughly match with existing or planned functionality. It's ok in some cases to have a "for example, <specific science example>" but these currently read much more like science use cases and generic analysis use cases -- too specific. I'd like to scale them back to generic analysis.

What we don't want is for users (or UC/SAC) to see this and assume our understanding of science use cases is very narrow and get worried about that.

I'll make a few examples but if you could review and generic-ify all analysis use cases.

Reply:
Confirmed the analysis use cases are generic, can apply to multiple science cases (i.e., aren't science-specific) and well matched to RSP functionality. These are great. Clicking resolved.